### DISCUSSION TITLE
Disable "User" View

### DISCUSSION THREAD

**Discussion by Biancabepunkt** [WeblateOrg/weblate#5902] [Q&A]
When working with external tranlators, given the rights only to tranlsate, I want to avoid that the "/user" pages are visible for these users due to data protection reasons. Is there a possiblity to disable the "User" view or deny the access for this group?

**Comment by nijel** [comment]
There is currently no way to completely block this. You can restrict access to signed-in users using https://docs.weblate.org/en/latest/admin/config.html#std-setting-LOGIN_REQUIRED_URLS
